### PR TITLE
Fix HTML for spec-prod

### PR DESCRIPTION
**Pull request by afs**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-entailment/pull/8.html" title="Last updated on Feb 9, 2023, 11:09 PM UTC (4ff6450)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-entailment/8/9100b37...4ff6450.html" title="Last updated on Feb 9, 2023, 11:09 PM UTC (4ff6450)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
         edDraftURI:           "https://w3c.github.io/sparql-entailment/spec/",
         testSuiteURI:         "https://w3c.github.io/rdf-tests/",
         shortName:            "sparql12-entailment",
-        copyrightStart:       "2008",
+        copyrightStart:       "2012",
```
Comment (gkellogg) [inline spec/index.html:20]: Does the copyright start change? I've left it as is in other specs, or was it simply wrong in this particular spec?

**Review by gkellogg — APPROVED** [review]
Looks fairly straightforward, and presumably is much the same in other specs. I did notice it choking on the `<a id="..."></a>` pattern, but `span` seems to work okay.

**Comment by gkellogg** [comment]
I'm afraid that to clear the Echidna error, you'll need to add yourself as Editor, or pick a victim :-).

File: spec/index.html (hunk 1 above)
Comment (afs) [reply spec/index.html:20]: This is a SPARQL 1.1 document. REC at 2013, FPWD sometime before.

2008 is SPARQL 1.0  (?probably should be FPWD date - 2008 is REC)

**Comment by afs** [comment]
@gkellogg wrote:
> I'm afraid that to clear the Echidna error, you'll need to add yourself as Editor, or pick a victim :-).

I'll override the merge as this is structural HTML fixing.

Unless/until there is an offer to be editor, this isn't viable as FPWD.

**Comment by gkellogg** [comment]
BTW, if you do add yourself as editor, you can annotate that using the `note` field to add something like "Interim Editor" so that it's clear your part is only temporary.

**Pull request merged**